packages/tezos-workers/tezos-workers.7.0~rc1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "tezos-tooling" { with-test & = version }
+  "ocamlfind" { build }
```

Comment:
Are these ocamlfind dependencies actually necessary?  (i.e. is there a Findlib or `ocamlfind` invocation from a dune rule?).  Dune includes the required ocamlfind functionality for the common case, so you may not need this.

Reply:
That's clearly a remaining of the past, thank you.